src/mainboard/seeed/beaglev/Makefile

```diff
@@ -0,0 +1,8 @@
+TARGET     = riscv64imac-unknown-none-elf
+QEMU       ?= qemu-system-riscv64
+QEMU_FLAGS += -m 1g -machine sifive_u,start-in-flash=true -nographic -device loader,addr=0x20000000,file=${IMAGE} -bios none -smp 4
```

Comment:
just put an empty task at the very top; every board is built in CI - the lists for skipping are only for formatting and crate testing. So something here like this shall do:

```Makefile
default:
    echo "Skipping Beagle V"
```